docs/antora-playbook.yml

```diff
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/asciidoctor/asciidoctor-intellij-plugin/main/src/main/resources/jsonSchemas/antoraPlaybookSchema.json
+
+site:
+  title: Reqstool Java Maven Plugin
```

Comment:
Or just Reqstool Maven Plugin - the Java is implied with Maven. Does not have to be the exact same as the artifact.

Reply:
Applied to Antora and pom.xml description as wel.

Reply:
alright. I'll change it.